app/main.py

```diff
@@ -1,6 +1,15 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args) -> Any:
+        if args in cache_dict:
+            print("Getting from cache")
+            return cache_dict[args]
```

Comment:
Don't repeat yourself, you can use "return cache_dict[args]" only 1 time, after if else construction :) This was stated in the checklist at the end of the "Readme" for this task.